### PR TITLE
[Translation] Crowdin is backing its translation bridge, thanks to them! \o/

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 7.1
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | -
| License       | MIT

Thank you Crowdin! /cc @andrii-bodnar

### CODE REVIEW

**Review by welcoMattic — APPROVED** [review]
🎉

File: src/Symfony/Component/Translation/Bridge/Crowdin/README.md [hunk 1]
```diff
@@ -23,7 +23,11 @@ where:
 Sponsor
 -------
 
-Help Symfony by [sponsoring][1] its development!
+This bridge for Symfony 5.4/6.0 is [backed][1] by [Crowdin][2].
```
Comment (xabbuh) [inline src/Symfony/Component/Translation/Bridge/Crowdin/README.md:26]: Shouldn't this have been 7.1?

Comment (nicolas-grekas) [reply src/Symfony/Component/Translation/Bridge/Crowdin/README.md:26]: indeed!

Comment (xabbuh) [reply src/Symfony/Component/Translation/Bridge/Crowdin/README.md:26]: fixed in eecb3bb2ae9b7db0c337cbf47e6e0f98822d5375

**Comment by xabbuh** [comment]
@SamiQu please stop spamming the repo

**Pull request merged**